### PR TITLE
Sync styles and themes for litmus-portal

### PR DESCRIPTION
**Pull request by AVRahul**
Signed-off-by: AVRahul <a.v.rahul11@gmail.com>

## Proposed changes
Have updated the theme file new label for styles in order to get the styles generic and be used only from the `themes.tsx` file. Also modified some color strings present inside components with a defined label.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/Settings/TeammingTab/Invitation/styles.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ const useStyles = makeStyles((theme: Theme) => ({
     alignItems: 'center',
     marginLeft: theme.spacing(2),
     justifyContent: 'space-between',
+    backgroundColor: theme.palette.common.white,
```
Comment (chandankumar4) [inline litmus-portal/frontend/src/views/Settings/TeammingTab/Invitation/styles.ts:21]: This need to change, do not mention the color name here, i.e: 'white'

**Review by chandankumar4 — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/pages/Settings/styles.ts [hunk 2]
```diff
@@ -2,14 +2,15 @@ import { makeStyles } from '@material-ui/core/styles';
...
+    color: 'inherit',
     display: 'flex',
     flexGrow: 1,
     maxWidth: '63.75rem',
     alignItems: 'center',
     justifyContent: 'space-between',
     marginLeft: theme.spacing(2),
     borderBottom: `1px solid ${theme.palette.customColors.black(0.1)}`,
-    backgroundColor: theme.palette.tabSelectorBackgroundColor,
+    backgroundColor: `${theme.palette.customColors.black(0)} !important`,
```
Comment (chandankumar4) [inline litmus-portal/frontend/src/pages/Settings/styles.ts:13]: please avoid using color with their name

Comment (AVRahul) [reply litmus-portal/frontend/src/pages/Settings/styles.ts:13]: yes, will update it, thanks

**Review by chandankumar4 — APPROVED** [review]
LGTM

**Review by arkajyotiMukherjee — APPROVED** [review]

File: litmus-portal/frontend/src/components/InfoFilled/index.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import formatCount from '../../utils/formatCount';
 import useStyles from './styles';
 
 interface CardValueData {
-  color: string;
+  color: string | undefined;
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/components/InfoFilled/index.tsx:10]: Isn't color a required field! It won't really work if its undefined right?

**Review by S-ayanide — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/components/InfoFilled/index.tsx (hunk 3 above)
Comment (AVRahul) [reply litmus-portal/frontend/src/components/InfoFilled/index.tsx:10]: yes, that's correct, so as in this case we have declared a custom variable in our themes file as `totalRunsCountColor?: string;` which accepts a string | undefined value, 
so, here in this case the color accepts undefined also as we have used the above-defined label here as one of the values for color.
Screenshot from 2020-10-06 18-39-10

Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/components/InfoFilled/index.tsx:10]: in PaletteOptions make `totalRunsCountColor: string`...remove the ? the error will go away

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/components/InfoFilled/index.tsx (hunk 3 above)
Comment (S-ayanide) [reply litmus-portal/frontend/src/components/InfoFilled/index.tsx:10]: You can make the `totalRunsCountColor` a required palette option, then you wouldn't need to use `undefined` in the interface.

Comment (AVRahul) [reply litmus-portal/frontend/src/components/InfoFilled/index.tsx]: yes, declaring totalRunsCountColor: string in palette option worked, Thanks @S-ayanide @arkajyotiMukherjee for the feedback.

**Review by S-ayanide — APPROVED** [review]

**Review by chandankumar4 — APPROVED** [review]
LGTM

**Review by arkajyotiMukherjee — APPROVED** [review]
LGTM

**Pull request merged**